runtime/vm/CRIUHelpers.cpp

```diff
@@ -200,7 +200,7 @@ juRandomReseed(J9VMThread *currentThread, void *userData, const char **nlsMsgFor
 	IDATA seedOffset = findinstanceFieldOffsetHelper(currentThread, hookRecord->instanceType, "seed", "Ljava/util/concurrent/atomic/AtomicLong;");
 	if (-1 != seedOffset) {
 #define JUCA_ATOMICLONG "java/util/concurrent/atomic/AtomicLong"
-		J9Class *jucaAtomicLongClass = hashClassTableAt(vm->systemClassLoader, (U_8 *)JUCA_ATOMICLONG, LITERAL_STRLEN(JUCA_ATOMICLONG));
+		J9Class *jucaAtomicLongClass = peekClassHashTable(currentThread, vm->systemClassLoader, (U_8 *)JUCA_ATOMICLONG, LITERAL_STRLEN(JUCA_ATOMICLONG));
```

Comment:
why is this changed?

Reply:
`hashClassTableAt ()` needs to be protected w/ `classTableMutex`, which `peekClassHashTable()` does.